app/util/calldata.test.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  getAddress,
+  Interface,
+  parseEther,
+  parseUnits,
+  zeroPadValue
+} from "ethers"
+
+import { decodeExecuteParams, decodeTransferParams } from "~app/util/calldata"
+import { AccountType } from "~packages/account"
+
+const accountAbi: string[] = [
+  "function transfer(address to, uint256 value) public returns (bool)"
+]
+const executeAbi: string[] = [
+  "function execute(address dest, uint256 value, bytes func) external"
+]
```

Comment:
Since these abi all need to be wrapped by `Interface`, you can directly wrap them here.

```suggestion
const transferInterface = new Interface([
  "function transfer(address to, uint256 value) public returns (bool)"
])
const executeInterface = new Interface([
  "function execute(address dest, uint256 value, bytes func) external"
])
```